.github/workflows/hil_test_trigger.yml

```diff
@@ -8,6 +8,10 @@ on:
     # Run workflow at the start of every day (12 AM UTC)
     - cron: "0 0 * * *"
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
```

Comment:
Could you elaborate in commit message what it does, why we want this change, what it solves?

Reply:
Updated! Thanks for keeping me honest 🙂